### DISCUSSION TITLE
VTOL servo mixer used as motor mixer

### DISCUSSION THREAD

**Discussion by vtol-kopter** [iNavFlight/inav#7386] [Q&A]
servo mixer are very flexible in latest INAV version. I like it.
motor mixer are not flexible enough to use it in a VTOL configuration, right?
I am building a VTOL delta plane with two rear motors and one front motor. The front motor is fixed and should work only on hover/copter mode. The two rear motors can be tilt by about 110deg. In "plane" position they work in the usual way just horizontally. In "Copter" position they work vertically with a small tilt to achieve yaw. 
It was easy to set up servo mixers for two independent servos to tilt the motors in the appropriate way. 
Now I looking for way to control motors itself in plane mode: 
- switch front motor off, 
- control rear motors just by RC thrust.
In plane mode I do not need any support from the FC, just pass thrust to motors. Flaps are controlled directly by the RC receiver.
In copter mode its just a tri-copter. I've already tested this mode, it works perfectly with the two rear motors achieving yaw.

My question is:
Can I configure a servo mixer/output so that it works just like the motor mixers? 
Do you have any suggestion for a completely different solution.

**Comment by vtol-kopter** [comment]
hello,
maybe my question is not clear enough, so I try it differently:
When I load the tricopter mixers I get for motor 1 mixer:
Throttle=1, Roll=0, Pitch=1,33, Yaw=0
When I set up a servo mixer with two lines like Stabilized Throttle = 100%, Stabilized Pitch = 133%,
does this mixer output the same signal as the predefined motor mixer ?
The goal is to switch off the motor by a logic condition. Is there another way to switch off a motor ?
thank you in advance for any support.
regards, Gerd

**Comment by a-marmot** [comment]
I have a similar question.  It would be nice to change the channel mix according to a variable aux input (rotor tilt in your case).  As the rotors change angle, yaw and roll switch according to sin, cos of the tilt angle.
